nodejs/nodejs-guestbook/skaffold.yaml

```diff
@@ -20,4 +19,16 @@ profiles:
 # use the cloudbuild profile to build images using Google Cloud Build
 - name: cloudbuild
   build:
-    googleCloudBuild: {}
\ No newline at end of file
+    googleCloudBuild: {}
+
+- name: buildpacks
+  build:
+    artifacts:
+    - image: node-guestbook-backend
```

Comment:
this should be `nodejs-guestbook-backend`

Reply:
Oh how annoying 😊